rfcs/0102-moderation-team.md

```diff
@@ -0,0 +1,94 @@
+---
+feature: moderation team
+start-date: 2021-08-18
+author: tomberek
+co-authors: blaggacao
+shepherd-team: (names, to be nominated and accepted by RFC steering committee)
+shepherd-leader: (name to be appointed by RFC steering committee)
+related-issues: https://github.com/NixOS/rfcs/pull/98
+---
+
+# Summary
+[summary]: #summary
+
+Establish a team to perform moderation.
+
+# Motivation
+[motivation]: #motivation
+
+There is not currently any official mechanism for moderation action. It's not
+sustainable to have to call on Graham any time there's a spammer or conflict
+that requires moderation, and we'd like to help the community become more
+self-regulating.
```

Comment:
I still think this motivation is a bit sparse. Can we include some specific instances where this team would have been helpful and what we did instead?

Reply:
Maybe an o-tone from @grahamc or @ryantm on how it becomes increasingly difficult to keep up with their self-assigned (and mostly unnoticeable) moderation work?

Reply:
I don't know what an "o-tone" is but this would be part of it, yes. The other part would be to clarify what portion of their moderation work actually *needs* doing, vs what portion can be left without official authority. It's plausible to me that all of it is in the former category, but these discussions are continually shrouded in lack of specification so I'm genuinely unsure.

Reply:
I think we need some documentation on the needs for additional structure around moderation, not the mere assertion. Perhaps there is some that I'm not aware of. I'm aware of the need for moderation in many areas, I'm a moderator elsewhere myself, but I haven't seen any moderation issues coming up in NixOS. If there have been, that simply means the current moderation is doing a great job. And I haven't seen them complaining about the burden. I haven't seen any stakeholders complaining about the moderation, either. Can we please get this documented?

Reply:
@ryantm Might you be able to step in and offer us a quotable paragraph that sums up your experience from your personal point of view? I also feel your or graham's input is missing.